### PR TITLE
make nav buttons larger and remove outline on back save and edit buttons

### LINKED ISSUE #91: Make nav buttons larger and remove outline on back, save and edit buttons
### Acceptance Criteria
- Home and monthly button should be slightly larger both width and height. Theres a line above them that is too close, it should also be slightly moved upwards 
- Back and Edit button on sub details should not be outlined
- Back and Save button on sub upsert should not be outlined
- A png of the page is attached to the Pull request (not the repository code)
- Looks good on a ~1080x2400 (google pixel 5)

### LINKED ISSUE #134: Details page change button's background to match the theme
The mark paid button and the delete button seems to be using the default bootstrap coloring. Make sure the mark paid button has the class "btn-primary" and "btn-danger" for the delete button. The colors should be the same as the buttons in the theme https://bootswatch.com/cyborg/

### PR DESCRIPTION
**Pull request by Penguine-Os**
Closes #91

### CODE REVIEW

**Review by chrisK00 — COMMENTED** [review]
Would you mind posting a screenshot of the home page? - after the upcoming changes

**Comment by chrisK00** [comment]
About the nav buttons
- Can we make the buttons close to each other again and centered so the user doesnt have to move their thumb to the other end of the screen
- Add some space to the bottom of the buttons, its quite easy to missclick and click the androids back/home/open apps buttons.
- Remove the -fill from the calendar icon so its consistent with the home button
- Also add a tiny bit of space above the icons (the line above them is too close)
- You can make the icons a tiny bit smaller
- When you push the icons from the upper divider or from the bottom, use padding on the buttons so that they are easily clicked

**Comment by Penguine-Os** [comment]
afbeelding

**Comment by chrisK00** [comment]
> afbeelding
> 

Space above looks good on this img. Is this be4/after my comment

File: subtrack.MAUI/Shared/NavBar.razor [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
     .custom-navbar {
-        height: 50px; /* Adjust the height value as needed */
+        height: 3rem; /* Adjust the height value as needed */
     }
 </style>
 
-<nav class="navbar navbar-expand bg-dark fixed-bottom justify-content-center custom-navbar" data-bs-theme="dark">
-    <div id="navbarColor02">
-        <ul class="navbar-nav">
+<nav class="navbar navbar-expand bg-dark fixed-bottom custom-navbar" data-bs-theme="dark">
```
Comment (chrisK00) [inline subtrack.MAUI/Shared/NavBar.razor]:
```suggestion
<nav class="navbar navbar-expand bg-dark fixed-bottom custom-navbar justify-content-center" data-bs-theme="dark">
```

File: subtrack.MAUI/Shared/NavBar.razor [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
+
+    <ul class="nav navbar-nav flex w-100 justify-content-around">
```
Comment (chrisK00) [inline subtrack.MAUI/Shared/NavBar.razor]:
```suggestion
    <ul class="navbar-nav">
```

File: subtrack.MAUI/Shared/NavBar.razor [hunk 3]
```diff
@@ -1,24 +1,21 @@
...
             <li class="nav-item">
                 <NavLink class="nav-link d-flex flex-column align-items-center" href="/" Match="NavLinkMatch.All">
-                    <i class="bi bi-house"></i>
-                    <span>Home</span>
-                </NavLink>
+                    <i class="bi bi-house" style="font-size: 1.75rem;"></i>
+                    </NavLink>
             </li>
             <li class="nav-item">
```
Comment (chrisK00) [inline subtrack.MAUI/Shared/NavBar.razor]:
```suggestion
            <li class="nav-item ms-4">
```

File: subtrack.MAUI/Shared/NavBar.razor [hunk 4]
```diff
@@ -1,24 +1,21 @@
...
                 <NavLink class="nav-link d-flex flex-column align-items-center" href="monthly" Match="NavLinkMatch.Prefix">
-                    <i class="bi bi-card-list"></i>
-                    <span>Monthly</span>
+                    <i class="bi bi-calendar2-week-fill" style="font-size: 1.75rem;"></i>
```
Comment (chrisK00) [inline subtrack.MAUI/Shared/NavBar.razor]:
```suggestion
                    <i class="bi bi-calendar" style="font-size: 1.65rem;"></i>
```

File: subtrack.MAUI/Shared/NavBar.razor [hunk 5]
```diff
@@ -1,24 +1,21 @@
...
-        <ul class="navbar-nav">
+<nav class="navbar navbar-expand bg-dark fixed-bottom custom-navbar" data-bs-theme="dark">
+
+    <ul class="nav navbar-nav flex w-100 justify-content-around">
             <li class="nav-item">
                 <NavLink class="nav-link d-flex flex-column align-items-center" href="/" Match="NavLinkMatch.All">
-                    <i class="bi bi-house"></i>
-                    <span>Home</span>
-                </NavLink>
+                    <i class="bi bi-house" style="font-size: 1.75rem;"></i>
```
Comment (chrisK00) [inline subtrack.MAUI/Shared/NavBar.razor]:
```suggestion
                    <i class="bi bi-house" style="font-size: 1.65rem;"></i>
```

File: subtrack.MAUI/Shared/NavBar.razor [hunk 6]
```diff
@@ -1,24 +1,21 @@
...
+                    </NavLink>
             </li>
             <li class="nav-item">
                 <NavLink class="nav-link d-flex flex-column align-items-center" href="monthly" Match="NavLinkMatch.Prefix">
```
Comment (chrisK00) [inline subtrack.MAUI/Shared/NavBar.razor]:
```suggestion
                <NavLink class="nav-link d-flex flex-column align-items-center pb-4 px-2" href="monthly" Match="NavLinkMatch.Prefix">
```

File: subtrack.MAUI/Shared/NavBar.razor [hunk 7]
```diff
@@ -1,24 +1,21 @@
...
 </style>
 
-<nav class="navbar navbar-expand bg-dark fixed-bottom justify-content-center custom-navbar" data-bs-theme="dark">
-    <div id="navbarColor02">
-        <ul class="navbar-nav">
+<nav class="navbar navbar-expand bg-dark fixed-bottom custom-navbar" data-bs-theme="dark">
+
+    <ul class="nav navbar-nav flex w-100 justify-content-around">
             <li class="nav-item">
                 <NavLink class="nav-link d-flex flex-column align-items-center" href="/" Match="NavLinkMatch.All">
```
Comment (chrisK00) [inline subtrack.MAUI/Shared/NavBar.razor]:
```suggestion
                <NavLink class="nav-link d-flex flex-column align-items-center pb-4 px-2" href="/" Match="NavLinkMatch.All">
```

File: subtrack.MAUI/Shared/NavBar.razor [hunk 8]
```diff
@@ -1,24 +1,21 @@
 ﻿<style>
     .custom-navbar {
-        height: 50px; /* Adjust the height value as needed */
+        height: 3rem; /* Adjust the height value as needed */
```
Comment (chrisK00) [inline subtrack.MAUI/Shared/NavBar.razor]:
```suggestion
        height: 4.1rem; /* Adjust the height value as needed */
```

**Review by chrisK00 — CHANGES_REQUESTED** [review]

**Comment by chrisK00** [comment]
qemu-system-x86_64_3cHNwuKScZ
After my suggested changes

**Comment by chrisK00** [comment]
closes #134

**Review by chrisK00 — APPROVED** [review]

**Pull request merged**